src/frontend/src/components/public/CommentsSidebar/index.tsx

```diff
@@ -31,14 +121,24 @@ const CommentSidebar = ({ comments }: CommentSidebarProps) => {
                   </button>
                   <ul className="dropdown-menu border-1 shadow-sm">
                     <li role="button">
-                      <button onClick={() => {}} className="dropdown-item">
+                      <button
+                        onClick={() => handleDeleteComment(comment.id)}
+                        className="dropdown-item"
+                        disabled={
+                          comment.createdByDisplayName !== loginUserName
+                        }
+                      >
                         Delete
                       </button>
                     </li>
                   </ul>
                 </div>
               </div>
-              <div>{comment.text}</div>
+              <div>
+                {comment.text !== null
```

Comment:
I think we discussed this yesterday in our group call. The check here should be for the `isActive` boolean instead of the for the text content itself.

Reply:
done